### PR TITLE
Blog post: Bachelor Thesis Katharina

### PR DESCRIPTION
**Pull request by kabu00002**
## Description 
Adding new blog post about the bachelor thesis of Katharina. 

## To dos
- [x] Blog post

## Status
- [x] Ready to go

### CODE REVIEW

**Review by PaulaKramer — COMMENTED** [review]
Thanks @kabu00002, I rephrased it a little bit. Can you add the date you submitted the thesis here? I added sept. 30, 2023 but I don't know if this is correct :D

File: content/blog/bachelor_thesis_katharina.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Katharina submitted her Bachelor thesis on 'Novel Kinase Ligand Generation using Subpocket-Based Docking'"
+date: 2023-09-30
```
Comment (kabu00002) [inline content/blog/bachelor_thesis_katharina.md:3]:
```suggestion
date: 2023-09-28
```

File: content/blog/bachelor_thesis_katharina.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Katharina submitted her Bachelor thesis on 'Novel Kinase Ligand Generation using Subpocket-Based Docking'"
+date: 2023-09-28
+author: katharina.buchthal
+---
+
+We are happy to annouce that Katharina successfully submitted her bachelor thesis 'Novel Kinase Ligand Generation using Subpocket-Based Docking'. Within this thesis, we developed an approach to automatically and efficiently generated novel kinase inhibitors using the structural information of the protein kinase target of interest. We make use of the prior knowledge about kinase ligands and the information on functional important subpockets/regions of highly conserved kinase binding pocket.
```
Comment (kabu00002) [inline content/blog/bachelor_thesis_katharina.md:7]:
```suggestion
We are happy to annouce that Katharina successfully submitted her bachelor thesis 'Novel Kinase Ligand Generation using Subpocket-Based Docking'. Within this thesis, we developed an approach to automatically and efficiently generated novel kinase inhibitors using the structural information of the protein kinase target of interest. We make use of the prior knowledge about kinase ligands and the information on functional important subpockets/regions of the highly conserved kinase binding pocket. 
```

File: content/blog/bachelor_thesis_katharina.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Katharina submitted her Bachelor thesis on 'Novel Kinase Ligand Generation using Subpocket-Based Docking'"
+date: 2023-09-28
+author: katharina.buchthal
+---
+
+We are happy to annouce that Katharina successfully submitted her bachelor thesis 'Novel Kinase Ligand Generation using Subpocket-Based Docking'. Within this thesis, we developed an approach to automatically and efficiently generated novel kinase inhibitors using the structural information of the protein kinase target of interest. We make use of the prior knowledge about kinase ligands and the information on functional important subpockets/regions of the highly conserved kinase binding pocket. 
+
+Katharina will continue her work as a student researcher at the Volkamer Lab to refine and improve the pipeline, e.g. adding a feature to enhance the diversity of generated ligands. In addition, we are working on a case study with our collaberation partners for experimental validation.
+
```
Comment (kabu00002) [inline content/blog/bachelor_thesis_katharina.md:10]:
```suggestion

[Here]({{< ref "/projects/kinfraglib_pocket_enum.md" >}}) is more information about the subpocket-based project.
```

File: content/blog/bachelor_thesis_katharina.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Katharina submitted her Bachelor thesis on 'Novel Kinase Ligand Generation using Subpocket-Based Docking'"
```
Comment (AndreaVolkamer) [inline content/blog/bachelor_thesis_katharina.md:2]:
```suggestion
title: "Katharina submitted her Bachelor thesis on kinase ligand generation'"
```

**Review by AndreaVolkamer — APPROVED** [review]

File: content/blog/bachelor_thesis_katharina.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Katharina submitted her Bachelor thesis on kinase ligand generation'"
```
Comment (AndreaVolkamer) [inline content/blog/bachelor_thesis_katharina.md:2]:
```suggestion
title: "Katharina submitted her Bachelor thesis on kinase ligand generation"
```

**Pull request merged**